src/m365/spo/commands/site/site-set.ts

```diff
@@ -201,6 +211,20 @@ class SpoSiteSetCommand extends SpoCommand {
     });
   }
 
+  private setGroupifiedSiteDescription(logger: Logger, args: CommandArgs): Promise<void> {
```

Comment:
As `description` is the only option that we need, let's simplify the command's signature to `description: string`